lib/mongoid/config.rb

```diff
@@ -137,6 +137,16 @@ module Config
     # See https://jira.mongodb.org/browse/MONGOID-5542
     option :prevent_multiple_calls_of_embedded_callbacks, default: true
 
+    # When this flag is true, callbacks for embedded documents will not be
+    # called. This is the default in 9.0.
```

Comment:
I feel like the name of the option might be backwards, if this is the case. I would expect `around_callbacks_for_embeds = true` to mean that around callbacks *are* invoked. If we want a true value to mean they are *not* invoked, perhaps the option should be called something like `omit_around_callbacks_for_embeds`?

Reply:
Updated, thank you!